packages/keycodes/src/index.js

```diff
@@ -181,28 +223,46 @@ export const displayShortcutList = mapValues( modifiers, ( modifier ) => {
 
 /**
  * An object that contains functions to display shortcuts.
- * E.g. displayShortcut.primary( 'm' ) will return '⌘M' on Mac.
  *
- * @type {WPKeycodeHandlerByModifier} Keyed map of functions to display
- *                                    shortcuts.
+ * @example
+ * ```
+ * // Assuming macOS:
+ * E.g. displayShortcut.primary( 'm' );
+ * // "⌘M"
+ * ```
+ *
+ * @type {WPModifierHandler<WPKeyHandler<string>>} Keyed map of functions to
+ *                                                 display shortcuts.
  */
 export const displayShortcut = mapValues(
 	displayShortcutList,
 	( shortcutList ) => {
-		return ( character, _isApple = isAppleOS ) =>
-			shortcutList( character, _isApple ).join( '' );
+		return /** @type {WPKeyHandler<string>} */ (
+			character,
+			_isApple = isAppleOS
+		) => shortcutList( character, _isApple ).join( '' );
 	}
 );
 
 /**
- * An object that contains functions to return an aria label for a keyboard shortcut.
- * E.g. shortcutAriaLabel.primary( '.' ) will return 'Command + Period' on Mac.
+ * An object that contains functions to return an aria label for a keyboard
+ * shortcut.
  *
- * @type {WPKeycodeHandlerByModifier} Keyed map of functions to shortcut ARIA
- *                                    labels.
+ * @example
+ * ```
```

Comment:
It might need a more precise type, in other places I see:

https://github.com/WordPress/gutenberg/blob/502bd52d559bd8914bdadfc1ec794618b46262d0/packages/plugins/src/components/plugin-area/index.js#L40

It's an improvement already as is though 👍

Reply:
In the VSCode overlay window these snippets look good and appear properly syntax highlighted. Is there another place we should check?

Reply:
In the generated documentation in README file it isn't wrapped with JS syntax highlighting.